chrome/browser/net/chrome_network_delegate.cc

```diff
@@ -402,10 +494,14 @@ int ChromeNetworkDelegate::OnBeforeStartTransaction(
       } else {
         AdblockPlus::FilterEngine::ContentType adblock_content_type;
 
+        // some some resource types we need to do element hiding as they take empty screen space
+        bool elemHide;
```

Comment:
could you please initialize it to `false`?

Reply:
Just for reference, https://en.cppreference.com/w/cpp/language/default_initialization

> otherwise, nothing is done: the objects with automatic storage duration (and their subobjects) are initialized to indeterminate values.

and examples below

> int x;               // OK: the value of x is indeterminate

Reply:
thanks for the link - that differs from java.

Reply:
the compiler should have thrown a warning (unless it has been disabled)

Reply:
fixed with explicitly setting of default value